spec/requests/provider_interface/audit_trail_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'Provider interface - audit trail', type: :request do
+  def create_application
+    application_form = create :application_form
+    application_choice = create(
```

Comment:
nit: We have a factory for this now: `create(:application_choice, :awaiting_provider_decision)`

Reply:
good point, I wasn't too happy with this messy setup